test/e2e/imds-v2-test

```diff
@@ -94,6 +94,7 @@ for i in `seq 1 10`; do
         DEPLOYED=1
         break
     fi
+    echo "Setup Loop $i/10, sleeping for 5 seconds"
```

Comment:
Could use `$SETUP_SLEEP` to be consistent with `$TAINT_CHECK_SLEEP`

Reply:
yeah, there's really no reason it can't be the same as the TAINT_CHECK_SLEEP for ultimate consistency